### PR TITLE
Remove a dependency on k8s.io/kubernetes

### PR DESCRIPTION
**Pull request by lufia**
k8s.io/kubernetes has **go.mod** file that contains invalid pseudo-version, v0.0.0, so Go toolchain will fail to update that module.

According to https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-505725449

> k8s.io/kubernetes is not primarily intended to be consumed as a module. Only the published subcomponents are (and go get works properly with those).

But we need types defined in **apis/stats**. Thus I considered two ways to resolve that:

a) add `replace` directive in our **go.mod**
b) copy files only we needed from **apis/stats**

I think the a) is awkward because there are many modules included by k8s.io/kubernetes but they are not used directly in mackerel-container-agent, and files we needed are very small quantities, so I chose the b).

### CODE REVIEW

File: local-apis/update.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -eu
+
+cd $(dirname $0)
```
Comment (astj) [inline local-apis/update.sh]: I prefer some tempdir to git clone to, but just IMO :D

**Comment by astj** [comment]
Should we use `./internal/` to explicitly prevent others like godoc.org from indexing this modules? (I'm not sure weather we should or not)

File: local-apis/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# local-apis
+
+Stats directory is generated from a part of k8s.io/kubernetes.
```
Comment (astj) [inline local-apis/README.md]: We might mention:
- what update.sh is for
- license term
  - Yes it's declared in types.go itself, but it's more kind.

**Comment by lufia** [comment]
> Should we use ./internal/

It's right. I will move there.

**Pull request merged**